_docs/concepts/what-is-istio/comparison.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Istio vs. Other Software
```

Comment:
Could we use "and" instead of "vs." in this doc? "vs." feels adversarial.

Maybe "Istio and Related Solutions"?

Reply:
Agreed, that sounds better